### ISSUE TITLE
Inverted class labels in IMDB fine-tuning

### ISSUE THREAD

**Issue by simopal6** [pytorch/fairseq#1082]
I followed the example for IMDB fine-tuning, but in the end it seems that the labels specified in the source files (0: negative, 1: positive) don't match the outputs of the model. The model achieves 0.96 training accuracy and 9.87 validation accuracy.

For example:
```
>>> roberta = RobertaModel.from_pretrained('checkpoints', 'checkpoint_last.pt', 'IMDB-bin')
>>> roberta.eval();
>>> tokens = roberta.encode("It's a terrible movie.");
>>> roberta.predict('sentence_classification_head', tokens)
tensor([[-5.4107e+00, -4.4784e-03]], grad_fn=<LogSoftmaxBackward>)
```
which is basically a prediction for class 1. Consistently, a positive review predicts class 0.
